src/components/team-session/component.tsx

```diff
@@ -126,7 +145,22 @@ const TeamSession: React.FC<Props> = ({ team }) => {
             <li />
           </ul>
 
-          <footer className="flex items-end p-2 border-t border-white border-opacity-10">
+          <footer className="flex flex-col items-end p-2 border-t border-white border-opacity-10">
+            {emojiPickerState && (
+              <Picker
+                theme="dark"
+                onSelect={handleAddReaction}
+                style={{ position: 'absolute', bottom: 0, left: "3rem", zIndex: 10 }}
+                title="Pick you emoji"
+              />
+            )}
+            <button
+              onClick={handleEmojiPickerOpen}
+              className="flex items-center justify-center w-full h-12 px-4 space-x-2 text-base text-white rounded-lg hover:bg-opacity-10 hover:bg-white"
+            >
+              <span role="img" aria-label="button">😁</span>
+              <p>Add an Emoji!</p>
```

Comment:
There should be a way of exiting the emoji modal, by pressing escape or by clicking outside the modal. Right now the only way to exit the modal is by adding an emoji.